### PR TITLE
Update intro to dapps developer doc

### PR DESCRIPTION
**Pull request by minimalsm**
## Description

This PR is part of a series to [audit the developer docs](https://github.com/ethereum/ethereum-org-website/issues/3908) in preparation for upgrades to our translation program.

## Link
Deploy preview - https://deploy-preview-4065--ethereumorg.netlify.app/en/developers/docs/dapps/

### CODE REVIEW

File: src/content/developers/docs/dapps/index.md [hunk 1]
```diff
@@ -15,41 +15,41 @@ Before learning about dapps, you should cover the [blockchain basics](/developer
...
 
-A dapp can have frontend code and user interfaces written in any language (just like an app) that can make calls to its backend. Furthermore, its frontend can be hosted on decentralized storage such as [IPFS](https://ipfs.io/).
+A dapp can have frontend code and user interfaces written in any language (just like an app) to make calls to its backend. Furthermore, its frontend can get hosted on decentralized storage such as [IPFS](https://ipfs.io/).
 
-- **Decentralized** means they are independent, and no one can control them as a group.
-- **Deterministic** i.e., they perform the same function irrespective of the environment they are executed.
-- **Turing complete**, which means given the required resources, the dapp can perform any action.
-- **Isolated**, which means they are executed in a virtual environment known as Ethereum Virtual Machine so that if the smart contract happens to have a bug, it won’t hamper the normal functioning of the blockchain network.
+- **Decentralized** - dapps are independent, and no one can control them as a group.
```
Comment (wackerow) [inline src/content/developers/docs/dapps/index.md:20]: Thought for a possible change here... Different dapps by themselves have different degrees of centralized vs. distributed control, but what they do have in common is existing on the same, decentralized network. 
```suggestion
- **Decentralized** - dapps operate on Ethereum, an open public decentralized platform where no one person or group has control
```

The way it currently reads seems misleading, as you can easily program a dapp that only a few people can control.

**Review by wackerow — CHANGES_REQUESTED** [review]
Left one comment while we're on this page, otherwise the cleanup looks good 👍🏻

File: src/content/developers/docs/dapps/index.md (hunk 1 above)
Comment (corwintines) [reply src/content/developers/docs/dapps/index.md:20]: This makes sense, not all projects are open source.

Comment (minimalsm) [reply src/content/developers/docs/dapps/index.md:20]: Nice catch and agreed.

**Pull request merged**